components/overledger/overledger.app.mjs

```diff
@@ -49,6 +87,7 @@ export default {
     }) {
       return this._makeRequest({
         method: "POST",
+        baseUrl: this._getBaseUrl(),
```

Comment:
This repetition is unnecessary, the method already receives the environment in the opts param

Reply:
within the "createHook" and "deletHook" methods do I just remove this param (baseUrl: this._getBaseUrl()) in the makeRequest methode call or do i replace here with sometthing else?

Reply:
> within the "createHook" and "deletHook" methods do I just remove this param (baseUrl: this._getBaseUrl()) in the makeRequest methods call or do i replace here with sometthing else? bare in mind these methods are used in the sources/triggers (new-contract-event-instant and watch-new-account-instant) and not actions sections

@philbuuza, Yes, please remove the baseUrl, but make sure you have added the environment reference in the props of the common source file and passed it as a parameter in both methods